v2/public/localscripts/scatter.js

```diff
@@ -1,4 +1,6 @@
-function scatter(selector, xdata, ydata, options) {
+function scatter(selector, xdata, ydata, options, labels) {
```

Comment:
Can we put the `labels` in the `options` object? I want to avoid having the argument list grow as we add new features (at some point we'll add `colors`, `sizes`, etc).